src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,69 @@
+package pages;
+
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+import org.openqa.selenium.support.PageFactory;
+
+public class LoginPage {
+
+    public LoginPage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    @FindBy(how = How.ID, using = "identifierId")
```

Comment:
You can use the @FindBy method like this as well:
@FindBy(id = "identifierId")
a bit shorter and easier to read.